pkg/bisect/bisect.go

```diff
@@ -167,6 +174,270 @@ func runImpl(cfg *Config, repo vcs.Repo, bisecter vcs.Bisecter, inst instance.En
 	return res, nil
 }
 
+//       Takes in config_bisect.pl output:
+//       Hmm, can't make any more changes without making good == bad?
+//       Difference between good (+) and bad (-)
+//        +DRIVER1=n
+//        +DRIVER2=n
+//        -DRIVER3=n
+//        -DRIVER4=n
+//        DRIVER5 n -> y
+//        DRIVER6 y -> n
+//       See good and bad configs for details:
+//       good: /mnt/work/linux/good_config.tmp
+//       bad:  /mnt/work/linux/bad_config.tmp
+func (env *env) parseConfigBisectLog(bisectLog []byte) []string {
```

Comment:
A random style comment: Go does not require forward declarations, so we generally start with more important functions at the top and move minor helpers to the bottom, this should definitely go below the main bisect function. This is not what readers should start reading this file from.

Reply:
Done